git-compat-util.h

```diff
@@ -1294,6 +1302,14 @@ static inline int is_missing_file_error(int errno_)
 #define flush_fscache() /* noop */
 #endif
 
+#ifndef getcache_fscache
+#define getcache_fscache() /* noop */
```

Comment:
```suggestion
#define getcache_fscache() (NULL)
```

... fixes the compile error on Linux for me.